libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -229,6 +229,7 @@
 #define HAL_FLOW_PX4_MAX_FLOW_PIXEL 4
 #define HAL_FLOW_PX4_BOTTOM_FLOW_FEATURE_THRESHOLD 30
 #define HAL_FLOW_PX4_BOTTOM_FLOW_VALUE_THRESHOLD 5000
+#define HAL_PARAM_DEFAULTS_PATH "/etc/arducopter/bebop.parm"
```

Comment:
I'd like to start thinking about standardizing this location. I think we need a dir in /etc that we can put everything. With the new build system we would even be able to "make install" to install the right files like other programs do.

I propose we use `/etc/apm` or `/etc/ardupilot`. Imagining more vehicles can be installed at the same time, but it's intended to run on the same board, I think `/etc/ardupilot/copter.parm` would be sufficient. What do you think?